test/elixir_internal_certificate/scraper/scrapers_test.exs

```diff
@@ -230,4 +212,237 @@ defmodule ElixirInternalCertificate.Scraper.ScrapersTest do
              ]
     end
   end
+
+  describe "get_user_search/1" do
```

Comment:
seem we are missing the test for `get_user_search/2`

Reply:
Good catch, updated in [ba8fc3c](https://github.com/nvminhtue/elixir_internal_certificate/pull/37/commits/ba8fc3c46630332abcc5a4027398965f1cbc898e)